temporalcli/commandsgen/commands.yml

```diff
@@ -4492,3 +4503,19 @@ option-sets:
         enum-values:
           - not_open
           - not_completed_cleanly
+
+  - name: workflow-update-options
+    options:
+      - name: versioning-override-behavior
+        type: string-enum
+        description: |
+          Override the versioning behavior of a Workflow.
+        required: true
+        enum-values:
+          - pinned
+          - auto_upgrade
+      - name: versioning-override-pinned-version
+        type: string
+        description: Override Pinned Version for a Worker Deployment (Only for pinned).
```

Comment:
This will need to change to match my PR https://github.com/temporalio/cli/pull/811 after it goes in, so you can do that now if you want or I can do it before we cut the release, either way is fine.